lib/streamlit/elements/button.py

```diff
@@ -29,20 +35,34 @@
 
 
 class ButtonMixin:
-    def button(self, label, key=None, help=None):
+    def button(
+        self,
+        label,
+        key: Optional[str] = None,
+        help: Optional[str] = None,
+        on_change: Optional[WidgetCallback] = None,
+        args: Optional[WidgetArgs] = None,
+        kwargs: Optional[WidgetKwargs] = None,
+    ) -> bool:
         """Display a button widget.
 
         Parameters
         ----------
         label : str
             A short label explaining to the user what this button is for.
-        key : str
+        key : Optional[str]
             An optional string to use as the unique key for the widget.
             If this is omitted, a key will be generated for the widget
             based on its content. Multiple widgets of the same type may
             not share the same key.
-        help : str
+        help : Optional[str]
             A tooltip that gets displayed when the button is hovered over.
+        on_change : Optional[Callable]
+            A callback invoked when the button is clicked.
```

Comment:
Should we add "an optional" to all optional arguments?

Reply:
:+1: done